vite.config.ts

```diff
@@ -0,0 +1,19 @@
+import { defineConfig } from 'vite';
+import svgrPlugin from 'vite-plugin-svgr';
+import reactRefresh from '@vitejs/plugin-react';
+import viteTsconfigPaths from 'vite-tsconfig-paths';
+
+export default defineConfig({
+  build: {
+    outDir: "build",
+  },
+  plugins: [
```

Comment:
Could you add the port number of the dev server as 3000 so that we can debug with the same settings as before?

```suggestion
  },
  server: {
    port: 3000,
  },
  plugins: [
```